include/mbedtls/aes.h

```diff
@@ -38,6 +38,7 @@
 
 #define MBEDTLS_ERR_AES_INVALID_KEY_LENGTH                -0x0020  /**< Invalid key length. */
 #define MBEDTLS_ERR_AES_INVALID_INPUT_LENGTH              -0x0022  /**< Invalid data input length. */
+#define MBEDTLS_ERR_AES_FEATURE_UNAVAILABLE               -0x006E  /**< Feature not available, e.g. unsupported AES key size. */
```

Comment:
#964 also adds an AES error code, they should be added consecutively. I think it would be best to use 0x0021 and 0x0023 as the new codes for AES, and #964 adds 0x0021, so this PR should add 0x0023.

Odd error codes _are_ allowed now, but their ranges are allocated separately from even codes. Although they are separates, it reduces confusion for humans to keep the ranges consecutive as much as possible. Think of the omission of odd codes as line numbers in BASIC — if we need to add more codes, we'll use the skipped values.